contracts/interfaces/ILoanCore.sol

```diff
@@ -7,19 +7,13 @@ import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import "../libraries/LoanLibrary.sol";
 
 import "./IPromissoryNote.sol";
-import "./IAssetWrapper.sol";
 import "./IFeeController.sol";
 import "./ILoanCore.sol";
 
 /**
  * @dev Interface for the LoanCore contract
  */
 interface ILoanCore {
-    /**
-     * @dev Emitted on initialization to share location of dependent notes
-     */
-    event Initialized(address collateralToken, address borrowerNote, address lenderNote);
```

Comment:
note: had to remove this to be able to use immutable keyword for these state variables. Big gas savings and this event was pretty useless anyways